src/components/collection/Picker.tsx

```diff
@@ -257,6 +259,34 @@ function CollectionPicker({ readOnly = false }: Props) {
                         onBlur={handlers.validateSelection}
                     />
                 )}
+                renderOption={(props, option, { selected }) => {
+                    return (
+                        <li {...props}>
+                            <Box
+                                sx={{
+                                    ml: -2,
+                                    mr: 0.5,
+                                }}
+                            >
+                                <Check
```

Comment:
What color is this icon? My browser dev tools are being a pain, but it _looks_ like the it is white in dark mode and black in light mode as opposed to being the primary text color set in the theme.

Reply:
They do match. Forced autocomplete open and they are set to the same color